kvmagent/kvmagent/plugins/ha_plugin.py

```
@@ -3289,15 +3289,30 @@ def _dump_ha_network_group_debug(self, stage, down_monitors=None, network_group_
 
+    @thread.AsyncThread
+    def _async_report_ha_network_group_status(self, url, host_uuid, network_group_status):
+        cmd = ReportHaNetworkGroupStatusCmd()
+        cmd.hostUuid = host_uuid
+        cmd.networkGroupStatus = network_group_status
+
+        try:
+            http.json_dump_post(url, cmd, {'commandpath': self.REPORT_HA_NETWORK_GROUP_STATUS_PATH})
+            logger.debug('reported ha network group status for host[%s], status:%s' % (host_uuid, network_group_status))
+            self._dump_ha_network_group_debug('report-status', network_group_status=network_group_status)
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)